apps/03TicTacToe.js

```diff
@@ -24,32 +24,81 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn)
+    || (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn)
+    || (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)) {
+    return true;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn)
+    || (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn)
+    || (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn)) {
+    return true;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn)
+    || (board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn)) {
+    return true;
```

Comment:
These win functions all look pretty good! Nice work. Did you think about returning `false` ever?